### DISCUSSION TITLE
How often does this project make a release?

### DISCUSSION THREAD

**Discussion by Wim-De-Clercq** [p1c2u/openapi-core#891] [General]
Hey,

I've been waiting since June for this fix here https://github.com/python-openapi/openapi-core/issues/852

But it's still not available on pypi.

When do releases happen?

**Comment by p1c2u** [comment]
Hi @Wim-De-Clercq  it's demand-based.  I will release it shortly. Thanks for the comment.

**Comment by Wim-De-Clercq** [comment]
Thank you!
